wagl/multifile_workflow.py

```diff
@@ -198,8 +199,8 @@ def run(self):
                       'water_vapour_dict': self.water_vapour,
                       'ozone_path': self.ozone_path,
                       'dem_path': self.dem_path,
-                      'brdf_path': self.brdf_path,
-                      'brdf_premodis_path': self.brdf_premodis_path}
+                      'brdf_path': self.brdf['brdf_path'],
```

Comment:
Question for @sixy6e : what happens here if the `brdf` dict has user-defined values?